lux/core/groupby.py

```diff
@@ -0,0 +1,65 @@
+import pandas as pd
+
+
+class LuxDataFrameGroupBy(pd.core.groupby.generic.DataFrameGroupBy):
+
+    _metadata = [
+        "_intent",
+        "_inferred_intent",
+        "_data_type",
+        "unique_values",
+        "cardinality",
+        "_rec_info",
+        "_min_max",
+        "_current_vis",
+        "_widget",
+        "_recommendation",
+        "_prev",
+        "_history",
+        "_saved_export",
+        "_sampled",
+        "_toggle_pandas_display",
+        "_message",
+        "_pandas_only",
+        "pre_aggregated",
+        "_type_override",
+    ]
+
+    def __init__(self, *args, **kwargs):
+        super(LuxDataFrameGroupBy, self).__init__(*args, **kwargs)
+
+    def aggregate(self, *args, **kwargs):
```

Comment:
Could you add some comments for each overridden function on the rationale on choosing to propogate metadata for some but not other (e.g. `filter`, `get_group`)? Is there a way to do this more uniformly across the functions?

Reply:
Will add some comments! I believe for `filter` and `get_group` we were more concerned about making sure `pre_aggregated` was not set to True. I believe the metadata was already being propagated through, but I'm not sure why so I'll look into this!

Reply:
Added comments! I realized I was definitely supposed to be propagating the metadata through `filter` and `get_group`, so I've added that. Thanks for the catch!